man/systemd-run.xml

```diff
@@ -92,6 +92,11 @@
     Consider using the <option>exec</option> service type (i.e. <option>--property=Type=exec</option>) to
     ensure that <command>systemd-run</command> returns successfully only if the specified command line has
     been successfully started.</para>
+
+    <para>After <command>systemd-run</command> passes the command to the manager, it performs variable
```

Comment:
"it" could refer to the manager or to systemd-run, that's unclear...

also manager → service manager